File: app/src/main/java/com/taipei/yanghaobo/kunu/db/Book.java
package com.taipei.yanghaobo.kunu.db;

/**
 * Google book object
 */
public class Book {
    private String mBookName;

    private String mBookAuthor;

    public Book(String bookName, String bookAuthor) {
        mBookName = bookName;
        mBookAuthor = bookAuthor;
    }

    public String getBookName() {
        return mBookName;
    }

    public String getBookAuthor() {
        return mBookAuthor;
    }
}

File: app/src/main/java/com/taipei/yanghaobo/kunu/db/Book.kt
package com.taipei.yanghaobo.kunu.db

import com.google.gson.annotations.SerializedName

/**
 * Google book object
 */
class Book(
  @field:SerializedName("title")
  val bookName: String, @field:SerializedName("authors")
  val bookAuthor: String
) {

  override fun toString(): String {
    return "Book{" +
        "mBookName='" + bookName + '\''.toString() +
        ", mBookAuthor='" + bookAuthor + '\''.toString() +
        '}'.toString()
  }
}
